packages/polaris-viz/src/components/TooltipContent/components/TooltipSeries/TooltipSeries.tsx

```diff
@@ -1,16 +1,31 @@
 import type {ReactNode} from 'react';
+import {Fragment} from 'react';
+import {useChartContext} from '@shopify/polaris-viz-core';
+
+import {TooltipSeriesName} from '../TooltipSeriesName';
 
 import styles from './TooltipSeries.scss';
 
-interface Props {
+export interface Props {
   children: ReactNode;
-  isEmpty?: boolean;
+  columnCount?: number;
```

Comment:
A `columnCount` prop feels a little too imperative, have you thought about making the props more declarative i.e. `hasPreview` and `hasTrend`? Then `TooltipSeries` can derive the column count based on those booleans